sys/arm64/arm64/elf_machdep.c

```diff
@@ -582,34 +580,34 @@ elf_reloc_internal(linker_file_t lf, char *relocbase, const void *data,
 #ifdef __CHERI_PURE_CAPABILITY__
 	case R_MORELLO_CAPINIT:
```

Comment:
I think we want to support R_MORELLO_CAPINIT at least in hybrid kernels, which using uintptr_t rather than uintcap_t gets in the way of.

Reply:
So at least for hybrid kernels we'd need to keep the extra methods... that or uintcap_t-ify everything :(

Reply:
Hmm, we don't support those today?  Is there a future use case for those in hybrid kernels (it looks like we are already not using caprelocs for Morello kernels)?